app/helpers/admin/breweries_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Admin::BreweriesHelper
+  def brewery_medium_logo(brewery)
+    return brewery.image.try(:url) ? brewery.image.medium_thumb.url : 'logo/logo-medium.png'
```

Comment:
__[rubocop]__
* Redundant `return` detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-explicit-return)
* Line is too long. [92/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)